src/components/views/rooms/MemberList.js

```diff
@@ -49,7 +49,7 @@ export default createReactClass({
         }
     },
 
-    componentWillMount: function() {
+    componentDidMount: function() {
```

Comment:
we assume `this._showPresence` is already stable by first render, it needs to happen in will-mount or statefully